developer-docs/how-to-guide/how_to_create_badge.md

```diff
@@ -16,178 +16,198 @@ This document explains the procedure of creating and awarding badges.
+rootOrgId: 0124758459210711040
+type: user
+subtype: certificate
+roles: BADGE_ISSUER
+images : C:\Users\Pictures\badge_logo.png
+```
+
+**Notes**
+  
+  * In the previous section we had created a new issuer entity which had been given the `issuerId` of `issuerslud-174` 
+  * When creating organisations and users, the system administrator received the `rootOrgId` for XYZ Corp as `0124758459210711040`
 
 
 #### Response Body
 
-	{
-	    "responseCode": "OK"
-	    "result": {
-	    	"badgeId": "badgeslug-66",
-	    	"criteria": "http://localhost:8000/public/badges/badgeslug-66/criteria",
-	    	"roles": ["BADGE_ISSUER"],
-	    	"description": "Badge Issuer",
-	    	"type": "user",
-	    	"rootOrgId": "0124758459210711040",
-			"issuerId": "issuerslug-174",
-	    	"createdDate": "2018-08-07T08:47:32.431314Z",
-	    	"recipientCount": 0,
-	    	"subtype": "certificate",
-	    	"issuerIdUrl": "http://localhost:8000/public/issuers/issuerslug-174",
-	    	"name": "Padma",
-	    	"badgeIdUrl": "http://localhost:8000/public/badges/badgeslug-66”
-	    }
-	}
- 
+```javascript
+{
+    "responseCode": "OK"
+    "result": {
+        "badgeId": "badgeslug-66",
+        "criteria": "http://localhost:8000/public/badges/badgeslug-66/criteria",
+        "roles": ["BADGE_ISSUER"],
+        "name": "Basic Alphabet Expert",
+        "description": "Awarded for expertise in using basic alphabet",
+        "type": "user",
+        "rootOrgId": "0124758459210711040",
+        "issuerId": "issuerslug-174",
+        "createdDate": "2018-08-07T08:47:32.431314Z",
+        "recipientCount": 0,
+        "subtype": "certificate",
+        "issuerIdUrl": "http://localhost:8000/public/issuers/issuerslug-174",
+        "badgeIdUrl": "http://localhost:8000/public/badges/badgeslug-66”
+    }
+}
+``` 
 
 1. Save the created ```badgeId```
 
-2. The badge ID created is required to [Award the badge](/how_to_create_badge#awarding-the-badge) to a user
+1. The badge ID created is required to [Award the badge](/how_to_create_badge#awarding-the-badge) to a user
 
 ### Awarding the Badge
 
-Once the badge class is created, the organization administrator issues the badge to employees as they finish the course. The badge Assertion API is associated with fetching and listing the assertions of a badge. 
+Once the badge class is created, any user with the `BADGE_ISSUER` role referenced when creating the badge class issues the badge to employees as they finish the course. The badge Assertion API is associated with fetching and listing the assertions of a badge. 
 
->Note: You would have saved the issuer ID received after creating the issuer and the badge ID received after creating the badge class. These IDs are required for awarding the badge.
+> Note: You would have saved the issuer ID received after creating the issuer and the badge ID received after creating the badge class. These IDs are required for awarding the badge.
```

Comment:
@Sunitashelar will also need authorization credentials for the user awarding the badge which is missing from the doc. Please connect with the developer to add these details

cc: @vrayulu @madhucr